### ISSUE TITLE
[EPIC] Solana support in mobile app

### ISSUE THREAD

**Issue by mnuky** [trezor/trezor-suite#10321]
Notes:
- Show tokens for sol and all testnet
- Refactor all thereum token related selectors, formatters and naming to generic token
- Adapt UI to work with solana similar to how it works with ethereum tokens (bonus points for making it work with ada as well)
- historical value - how? 
- graph - also how? From txns? waiting on Dan to finish txn storage?
- check that token definitions work
- NFTs, cNFTs? (not now)

```[tasklist]
### Tasks
- [ ] https://github.com/trezor/trezor-suite/issues/10236
```

**Comment by mnuky** [comment]
closing the epic - we will no longer use issues as epic, instead Milestones will be used
